pkg/configtx/policies.go

```diff
@@ -146,20 +143,10 @@ func (c *ConfigTx) RemoveApplicationOrgPolicy(orgName, policyName string) error
 	return nil
 }
 
-// AddConsortiumOrgPolicy modifies an existing organization in a consortiums configuration's policies.
-// When the policy exists it will overwrite the existing policy.
-func (c *ConfigTx) AddConsortiumOrgPolicy(consortiumName, orgName, policyName string, policy Policy) error {
-	groupKey := ConsortiumsGroupKey
-
-	consortiumGroup, ok := c.updated.ChannelGroup.Groups[groupKey].Groups[consortiumName]
-	if !ok {
-		return fmt.Errorf("consortium '%s' does not exist in channel config", consortiumName)
-	}
-
-	orgGroup, ok := consortiumGroup.Groups[orgName]
-	if !ok {
-		return fmt.Errorf("%s org '%s' does not exist in channel config", strings.ToLower(groupKey), orgName)
-	}
+// SetConsortiumOrgPolicy modifies an existing organization in a consortiums configuration's policies.
+// If a policy already exist in current configuration, its value will be overwritten.
```

Comment:
Let's be consistent with how we write the comments for application and orderer org policy

Reply:
done